model/src/test/utils.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+"""
+test utilities
+"""
+
+import numpyro.distributions as dist
+import pyrenew.transformation as t
+from pyrenew.metaclass import DistributionalRV, TransformedRandomVariable
+from pyrenew.process import SimpleRandomWalkProcess
+
+
+def get_default_rt():
```

Comment:
suggest renaming. Something like "simple_rt()". Also might be useful to have arguments for names (with the same defaults as here). That way you could use the function twice in the same model if you needed to.

Reply:
changed in 875e28392516bc6ba06fb2de210368a4dfad68fe